data/tlg2200/tlg001/tlg2200.tlg001.opp-grc1.xml

```diff
@@ -65,12 +65,11 @@
                             <author ref="urn:cts:greekLit:tlg2200">Libanius</author>  
                             <editor>
                                 <persName>
-                                    <name xml:lang="lat">Richardus Foerster</name>
-                                    <addName xml:lang="deu">Richard Foerster</addName>
+                                    <name>Richard Foerster</name>
                                 </persName>
                             </editor>                           
-                            <title type="main">Libanii Opera</title>
-                            <title type="subordinate">Epistulae 1-839</title>
+                            <title type="main" xml:lang="lat">Libanii Opera</title>
+                            <title type="subordinate" xml:lang="lat">Epistulae 1-839</title>
```

Comment:
There's an inconsistency with `<title type="subordinate"...>` versus `<title type="sub"...>`. You mainly use the latter.

Reply:
I think sub is actually better, and will change that and try to stick with it in the future.